src/components/Chart/Bars.js

```diff
@@ -747,7 +748,9 @@ export const propTypes = {
   inlineLabelPosition: PropTypes.string,
   tLabel: PropTypes.func.isRequired,
   description: PropTypes.string,
-  showBarValues: PropTypes.bool
+  showBarValues: PropTypes.bool,
+  skipYLabels: PropTypes.bool,
```

Comment:
could this be auto computed when `props.color === props.y` and auto turn on color legend?

Reply:
and when there is more than one set of data (like in the example)